Basic/ClientDriven/ProjectSettings/GraphicsSettings.asset

```diff
@@ -3,7 +3,7 @@
 --- !u!30 &1
 GraphicsSettings:
   m_ObjectHideFlags: 0
-  serializedVersion: 14
+  serializedVersion: 16
```

Comment:
Did the graphics settings changes come automatically through the package upgrade?